### PR TITLE
2020 changes

### PR DESCRIPTION
**Pull request by andyneff**
- Update vsi_common
- Improve just targets for starting redis
- Make redis hostname a separate variable, so that is can be on a different node
- Remove `:rw` flags on redis mounts, to cope with bug in certain versions of redis
- Added NoStackException, so you can throw an exception that doesn't print a stack trace, just ends the program execution
- Add traceback to celery worker exceptions, to make debugging easier.
- Handle decorated "tasks"
- Various bug fixes

### CODE REVIEW

File: terra/logger.py [hunk 1]
```diff
@@ -71,9 +71,13 @@
 import struct
 import select
 import pickle
+import functools
```
Comment (drewgilliam) [inline terra/logger.py:74]: Is `functools` used?

File: terra/logger.py [hunk 2]
```diff
@@ -71,9 +71,13 @@
...
 
 import terra
-from terra.core.exceptions import ImproperlyConfigured
+from terra.core.exceptions import (
+  ImproperlyConfigured, NO_STACK_EXCEPTIONS, setup_logging_exception_hook,
```
Comment (drewgilliam) [inline terra/logger.py:78]: is `NO_STACK_EXCEPTIONS` used?

**Comment by drewgilliam** [comment]
Also note - pep8 failed, and there are CI warnings about redis secret files:
`WARNING: Service "redis-commander" uses an undefined secret file "/root/repo/redis_commander_password.secret", the following file should be created "/root/repo/redis_commander_password.secret"`

File: terra/logger.py (hunk 1 above)
Comment (andyneff) [reply terra/logger.py:74]: Nope, flask8 caught that (and a few others) too ;)

**Comment by andyneff** [comment]
I see that warning... For now, it's harmless and I'm going to ignore it. I'll see if I can come up with a "good" solution to bypass it in the future.

**Review by drewgilliam — APPROVED** [review]

**Pull request merged**